### PR TITLE
Add Prettier, husky and lint-staged for auto code formatting

### PR DESCRIPTION
**Pull request by a-kriya**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,27 +44,32 @@
     "@typescript-eslint/eslint-plugin": "^4.18.0",
     "@typescript-eslint/parser": "^4.18.0",
     "@vitejs/plugin-vue": "1.x.x",
+    "@vue/cli-plugin-babel": "^4.5.13",
```
Comment (a-kriya) [inline package.json:47]: With this added, I was able to successfully run `npm run test:unit`.

Would this solve the issue from your open PR, @mariusa?

File: docs/components/Progress.md [hunk 2]
```diff
@@ -201,45 +204,45 @@ The striped gradient can also be animated by setting the `animated` prop.
...
 #### Slots
 
-| Name  | Description |
-| --- | --- |
-| `default`  | Content to place in the progress bar. Overrides the `label`, `label-html`, `show-progress` and `show-value` props |
+| Name      | Description                                                                                                       |
+| --------- | ----------------------------------------------------------------------------------------------------------------- |
+| `default` | Content to place in the progress bar. Overrides the `label`, `label-html`, `show-progress` and `show-value` props |
 
 <script>
```
Comment (a-kriya) [inline docs/components/Progress.md:247]: Is this supposed to be here?

Comment (cdmoro) [reply docs/components/Progress.md:247]: What do you mean with 'this'? The changes in Progress.md look just fine

Comment (a-kriya) [reply docs/components/Progress.md:247]: The script block. Seems a bit random: https://github.com/cdmoro/bootstrap-vue-3/blob/main/docs/components/Progress.md

Comment (cdmoro) [reply docs/components/Progress.md:247]: The scrip block is necessary to show the animation state, to achieve this behavior https://getbootstrap.com/docs/5.1/components/progress/#animated-stripes (see the "Toogle animation" button)

**Pull request merged**